.gitattributes

```diff
@@ -1,5 +1,5 @@
-*.jar filter=lfs diff=lfs merge=lfs -text
-*.dat filter=lfs diff=lfs merge=lfs -text
-*.zip filter=lfs diff=lfs merge=lfs -text
-*.gz filter=lfs diff=lfs merge=lfs -text
-*.bz2 filter=lfs diff=lfs merge=lfs -text
+# *.jar filter=lfs diff=lfs merge=lfs -text
```

Comment:
disabling these for now so that no new files are added to lfs.

Reply:
"for now" -- and how will it be in the long term?

Reply:
If we no longer need lfs, how about deleting rather than commenting out?

Reply:
sure, I will just delete it.

Reply:
> [x] push to production needs to move from ant to maven

done